service/adapters/pubsub/request_pubsub.go

```diff
@@ -7,9 +7,9 @@ import (
 )
 
 type Request struct {
-	Ctx context.Context
-	Rw  rpc.ResponseWriter
-	Req *rpc.Request
+	Ctx    context.Context
+	Stream rpc.Stream
```

Comment:
Renamed `rpc.ResponseWriter` to `rpc.Stream` as this now carries data in both directions.